mongoengine/base/document.py

```diff
@@ -193,9 +195,20 @@ def validate(self, clean=True):
             message = "ValidationError (%s:%s) " % (self._class_name, pk)
             raise ValidationError(message, errors=errors)
 
-    def to_json(self):
+    def to_json(self, json_options=None):
```

Comment:
This can stay as-is for now, but FYI I don't think we use the `to_json` method nor care to use it in the future.

While it could be _mildly_ helpful to have some generic built-in serialization, many applications have their own subtly different rules for how to serialize an object, and thus they their own functions (or e.g. Pydantic models) to define that serialization's exact behavior.

I'd be fine with getting rid of this method entirely at some point.

Reply:
Pretty sure I ran into a good number of test failures when confirming this was required - I'm running the suite against them right now and will report that back in the morning.

Reply:
Yep, we hit it from a few different areas (I'll send the report in a private channel).